src/directives/google-map.ts

```diff
@@ -191,10 +191,12 @@ export class SebmGoogleMap implements OnChanges,
     });
   }
 
+  get wrapper(): GoogleMapsAPIWrapper { return this._mapsWrapper; }
```

Comment:
Yeah, you have to place your custom component inside the <sebm-google-map> compontent. This  is a conscious design decision.